Articles/Section-Trees/tree-actions.md

```diff
@@ -0,0 +1,86 @@
+# Tree Actions
+
+Items in an Umbraco Tree can have associated Actions, the actions visible to the currently logged in user can be controlled via User Permissions.
```

Comment:
Think this is two sentences - i.e. "... can have associated Actions.  The actions visible...."